src/calculateRentalCost.js

```diff
@@ -4,7 +4,22 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const BASE_PRICE_PER_DAY = 40;
+  const BASIC_DISCOUNT = 20;
+  const LONG_TERM = 7;
+  const LONG_TERM_DISCOUNT = 50;
+
+  const basePrice = days * BASE_PRICE_PER_DAY;
+
+  if (days >= LONG_TERM) {
+    return basePrice - LONG_TERM_DISCOUNT;
+  }
+
+  if (days >= 3) {
+    return basePrice - BASIC_DISCOUNT; //
```

Comment:
The current logic applies the basic discount only if the number of days is 3 or more and less than 7. If the number of days is 7 or more, both discounts should be applied according to the task description. Consider adjusting the logic to accommodate this requirement.